aws/resource_aws_wafv2_rule_group_test.go

```diff
@@ -1425,6 +1425,24 @@ func TestAccAwsWafv2RuleGroup_XssMatchStatement(t *testing.T) {
 	})
 }
 
+func testAccPreCheckAWSWafv2(t *testing.T) {
```

Comment:
thinking here 🤔 since this tests only `regional` scope (though I'd assume `cloudfront` would error in the same way for govcloud/we don't make use of it in our tests atm), it could help to capture this in the func name?

Reply:
i was considering passing the scope here as a param but feels unnecessary/could be refactored in the future if a case presents itself

Reply:
Sounds good! Fixed up.